sanic/app.py

```diff
@@ -724,7 +724,7 @@ async def handle_exception(
         A handler that catches specific exceptions and outputs a response.
 
         :param request: The current request object
-        :type request: :class:`SanicASGITestClient`
+        :type request: Request
```

Comment:
Can you keep the class reference so that Sphinx will create a link?

https://stackoverflow.com/a/21297186/1036843

Reply:
You can run the following to standup the API docs locally:

```
make docs-serve
```

Reply:
As we discussed, sphinx can auto detect the type-hints and link the doc to the class reference. Can you try it on your end and test this feature?